experimental/adaptive-tool/README.md

```diff
@@ -1,57 +1,118 @@
-- completion 
-![lu_completion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/lu_completion.gif)
+#### LU documents
+- Syntax highlighting
+![lu_syntax_highlighting](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lu_syntax_highlighting.png)
+- Diagnostic check
+![lu_diagnostic](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lu_diagnostic.png)
+- Completion 
+![lu_completion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lu_completion.gif)
+
+#### LG documents
+- Syntax highlighting
+![lg_syntax_highlighting](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lg_syntax_highlighting.png)
+
+- Diagnostic check
+![lg_diagnostic](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/lg_diagnostic.gif)
+
+- Template reference hover
+![template_hover](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/template_hover.png)
+
+- Builtin function hover
+![function_hover](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/function_hover.png)
+
+- Builtin function and template suggestion
+![function_template_suggestion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/function_template_suggestion.gif)
+
+- Structure property suggestion
+![structure_suggestion](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/structure_suggestion.gif)
 
-#### .lg documents
-- syntax highlighting
-![lg_syntax_highlighting](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/lg_syntax_highlighting.png)
+- Template definition
+![template_definition](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/main/experimental/adaptive-tool/resources/images/template_definition.gif)
 
-- diagnostic check
-![lg_diagnostic](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/lg_diagnostic.gif)
+## Debugging Feature
+### Setting up and using Visual Studio Code to run client and server
+#### Setting up
+To configure Visual Studio Code you need to add a target in your launch.settings file.
 
-- template reference hover
-![template_hover](https://raw.githubusercontent.com/microsoft/BotBuilder-Samples/master/experimental/adaptive-tool/resources/images/template_hover.png)
+* **Bot: Launch language server and client on vscode** - Configuration for building and launching your client on vscode and connecting to server
+Example is:
+```json
```

Comment:
I think those two configuration files is not for debugging .dialog or .lg, right?